### PR TITLE
trigger auto-import after removing the disable-auto-import label

### PR DESCRIPTION
**Pull request by elgnay**
The related issue: https://issues.redhat.com/browse/ACM-11477

### CODE REVIEW

File: pkg/controller/autoimport/manager.go [hunk 1]
```diff
@@ -88,6 +92,37 @@ func Add(ctx context.Context,
...
+		},
+		predicate.Predicate(predicate.Funcs{
+			GenericFunc: func(e event.GenericEvent) bool { return false },
+			DeleteFunc:  func(e event.DeleteEvent) bool { return false },
+			CreateFunc:  func(e event.CreateEvent) bool { return false },
+			UpdateFunc: func(e event.UpdateEvent) bool {
+				// handle the removal of the disable-auto-import annotation
+				_, oldAutoImportDisabled := e.ObjectOld.GetAnnotations()[apiconstants.DisableAutoImportAnnotation]
+				_, newAutoImportDisabled := e.ObjectNew.GetAnnotations()[apiconstants.DisableAutoImportAnnotation]
+				if oldAutoImportDisabled && !newAutoImportDisabled {
```
Comment (zhujian7) [inline pkg/controller/autoimport/manager.go:116]:
```suggestion
				return oldAutoImportDisabled && !newAutoImportDisabled
```
nit

Comment (elgnay) [reply pkg/controller/autoimport/manager.go]: Updated

**Pull request merged**